doc/tutorial_regression.rst

```diff
@@ -170,106 +182,155 @@ by the Jackknife+, Jackknife-minmax, CV+, and CV-minmax strategies.
             title=strategy
         )
 
-.. image:: images/tuto_2.png
-    :align: center
+
+
+.. image:: tutorial_regression_nb_files/tutorial_regression_nb_16_0.png
+
 
 At first glance, the four strategies give similar results and the
 prediction intervals are very close to the true confidence intervals.
 Let’s confirm this by comparing the prediction interval widths over
 :math:`x` between all strategies.
 
-.. code:: python
+.. code:: ipython3
 
     fig, ax = plt.subplots(1, 1, figsize=(7, 5))
+    ax.axhline(1.96*2*noise, ls="--", color="k", label="True width")
     for strategy in STRATEGIES:
-        ax.plot(X_test, y_pis[strategy][:, 1, 0] - y_pis[strategy][:, 0, 0])
-    ax.axhline(1.96*2*noise, ls="--", color="k")
+        ax.plot(X_test, y_pis[strategy][:, 1, 0] - y_pis[strategy][:, 0, 0], label=strategy)
     ax.set_xlabel("x")
     ax.set_ylabel("Prediction Interval Width")
-    ax.legend(strategies + ["True width"], fontsize=8)
+    _ = ax.legend(fontsize=10, loc=[1, 0.4])
 
-.. image:: images/tuto_3.png
-    :align: center
 
 
-As expected, the prediction intervals estimated by the Naive method
-are slightly too narrow. The Jackknife, Jackknife+, CV, and CV+ give
-similar widths that are very close to the true width. On the other hand,
-the widths estimated by Jackknife-minmax and CV-minmax are slightly too
-wide. Note that the widths given by the Naive, Jackknife, and CV strategies
-are constant since the prediction intervals are estimated upon the
-residuals of the training data only.
+.. image:: tutorial_regression_nb_files/tutorial_regression_nb_18_0.png
+
+
+As expected, the prediction intervals estimated by the Naive method are
+slightly too narrow. The Jackknife, Jackknife+, CV, CV+, JaB, and J+aB
+give similar widths that are very close to the true width. On the other
+hand, the widths estimated by Jackknife-minmax and CV-minmax are
+slightly too wide. Note that the widths given by the Naive, Jackknife,
+and CV strategies are constant since the prediction intervals are
```

Comment:
En fait c'est parce que "there is a single model used for prediction, perturbed models are ignored at prediction time"

Reply:
Oui, tu as raison.